src/Categories/Category/Construction/mu-Bialgebras.agda

```diff
@@ -0,0 +1,63 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Category.Construction.mu-Bialgebras where
+
+open import Level
+open import Function using (_$_)
+
+open import Categories.Category
```

Comment:
Could you please put more `using` clauses on your `import`s?

Reply:
If I use more than 3 things from a module it becomes a kind of egg-hunt to post hoc add them all as qualified imports (add members, check errors, repeat), and even more annoying to do it as you go (if you want to use a new member, you always need to go back and add it to the import list, also, you need to remove members you no longer use).
I think it would really help this project (and agda development in general) if there was language support for import management as there is in e.g. Haskell, with `haskell-language-server`. I opened an issue on `agda-unused` to ask why module applications are such a stumbling block, but haven't heard back yet: https://github.com/msuperdock/agda-unused/issues/1 .

Reply:
I agree, it's a big pain. It should be a feature of Agda.